cloud-aws/src/main/java/com/sequenceiq/cloudbreak/cloud/aws/connector/resource/AwsRdsStartService.java

```diff
@@ -0,0 +1,53 @@
+package com.sequenceiq.cloudbreak.cloud.aws.connector.resource;
+
+import com.amazonaws.services.rds.AmazonRDS;
+import com.amazonaws.services.rds.model.StartDBInstanceRequest;
+import com.sequenceiq.cloudbreak.cloud.aws.AwsClient;
+import com.sequenceiq.cloudbreak.cloud.aws.scheduler.AwsBackoffSyncPollingScheduler;
+import com.sequenceiq.cloudbreak.cloud.aws.task.AwsPollTaskFactory;
+import com.sequenceiq.cloudbreak.cloud.aws.view.AwsCredentialView;
+import com.sequenceiq.cloudbreak.cloud.context.AuthenticatedContext;
+import com.sequenceiq.cloudbreak.cloud.exception.CloudConnectorException;
+import com.sequenceiq.cloudbreak.cloud.task.PollTask;
+import org.springframework.stereotype.Service;
+
+import javax.inject.Inject;
+import java.util.concurrent.ExecutionException;
+import java.util.concurrent.TimeoutException;
+
+@Service
+public class AwsRdsStartService {
+
+    private static final String SUCCESS_STATUS = "available";
+
+    @Inject
+    private AwsClient awsClient;
+
+    @Inject
+    private AwsPollTaskFactory awsPollTaskFactory;
+
+    @Inject
+    private AwsBackoffSyncPollingScheduler<Boolean> awsBackoffSyncPollingScheduler;
+
+    public void start(AuthenticatedContext ac, String dbInstanceIdentifier) throws ExecutionException, TimeoutException, InterruptedException {
+        AwsCredentialView credentialView = new AwsCredentialView(ac.getCloudCredential());
+        String regionName = ac.getCloudContext().getLocation().getRegion().value();
+        AmazonRDS rdsClient = awsClient.createRdsClient(credentialView, regionName);
```

Comment:
some log would be nice that we would like start the which db instance

Reply:
Done in https://github.com/hortonworks/cloudbreak/blob/fbe8818ef58b368634efdc3e7e16b282435014b9/redbeams/src/main/java/com/sequenceiq/redbeams/service/stack/RedbeamsStartService.java#L41

Should I implement it here as well?